### ISSUE TITLE
Outdated transitive nom dependencies

### ISSUE THREAD

**Issue by nashley** [blacknon/hwatch#95]
```
  Downloaded hwatch v0.3.10
...
    Finished release [optimized] target(s) in 1m 23s
warning: the following packages contain code that will be rejected by a future version of Rust: nom v4.2.3, nom v5.1.2
note: to see what the problems were, use the option `--future-incompat-report`, or run `cargo report future-incompatibilities --id 1`
```

`cargo tree` shows the outdated `nom` dependencies coming from `ansi-parser v0.8.0` and `termwiz v0.15.0`'s `terminfo v0.7.3` respectively:
```
hwatch v0.3.10 (/home/user/git/hwatch)
├── ansi-parser v0.8.0
...
│   └── nom v4.2.3
...
└── termwiz v0.15.0
...
    ├── terminfo v0.7.3
    │   ├── nom v5.1.2
...
```

[`ansi-parser`](https://gitlab.com/davidbittner/ansi-parser/-/blob/master/Cargo.toml) doesn't seem to have an updated version available, but [`termwiz`](https://github.com/wez/wezterm/blob/main/termwiz/Cargo.toml) does (and it uses an updated `terminfo` that uses `nom` `7`)

**Comment by blacknon** [comment]
Wow, Thanks!
I will update it in the next version.

**Comment by blacknon** [comment]
The issue was resolved by applying version 0.9.0 of ansi-parser.
This issue will be closed. Thank you for the issue.
